samples/dml.js

```diff
@@ -474,32 +474,46 @@ function writeWithTransactionUsingDml(instanceId, databaseId, projectId) {
         // Note: Cloud Spanner interprets Node.js numbers as FLOAT64s, so they
         // must be converted (back) to strings before being inserted as INT64s.
 
-        transaction.runUpdate({
-          sql: `UPDATE Albums SET MarketingBudget = @Budget
-            WHERE SingerId = 1 and AlbumId = 1`,
-          params: {
-            Budget: firstBudget.toString(),
-          },
-        });
-        transaction.runUpdate({
-          sql: `UPDATE Albums SET MarketingBudget = @Budget
-            WHERE SingerId = 2 and AlbumId = 2`,
-          params: {
-            Budget: secondBudget.toString(),
-          },
-        });
+        return transaction
+          .runUpdate({
+            sql: `UPDATE Albums SET MarketingBudget = @Budget
+                WHERE SingerId = 1 and AlbumId = 1`,
+            params: {
+              Budget: firstBudget,
+            },
+          })
+          .then(() =>
+            transaction.runUpdate({
+              sql: `UPDATE Albums SET MarketingBudget = @Budget
+                    WHERE SingerId = 2 and AlbumId = 2`,
+              params: {
+                Budget: secondBudget,
+              },
+            })
+          );
       })
       .then(() => {
-        // Commits the transaction and send the changes to the database
-        return transaction.commit();
+        // Get updated albums' budgets.
+        const query = `SELECT MarketingBudget FROM Albums
+          WHERE (SingerId = 1 AND AlbumId = 1) OR (SingerId = 2 AND AlbumId = 2)
+          ORDER BY SingerId`;
+        return transaction.run(query);
       })
-      .then(() => {
-        console.log(
-          `Successfully executed read-write transaction using DML to transfer ${transferAmount} from Album 2 to Album 1.`
-        );
+      .then(results => {
+        // Verify the accuracy of the new budgets.
+        const rows = results[0].map(row => row.toJSON());
+        if (
+          rows[0].MarketingBudget === firstBudget &&
+          rows[1].MarketingBudget === secondBudget
+        ) {
+          console.log(
+            `Successfully executed read-write transaction using DML to transfer ${transferAmount} from Album 2 to Album 1.`
+          );
+        }
```

Comment:
We can't really have this part because this would make the Node.js sample inconsistent with that of the other languages. Is it ok if we remove it?

Reply:
https://cloud.google.com/spanner/docs/getting-started/go#update-data for example.

Reply:
Sure.

It was added to catch/verify that the current issue would have been caught.

Done!